content-resources/src/main/java/fi/nls/oskari/util/UserDataStyleMigrator.java

```diff
@@ -0,0 +1,161 @@
+package fi.nls.oskari.util;
+
+import fi.nls.oskari.db.DatasourceHelper;
+import fi.nls.oskari.domain.map.wfs.WFSLayerOptions;
+import org.json.JSONObject;
+
+import javax.sql.DataSource;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.HashMap;
+import java.util.Map;
+
+public class UserDataStyleMigrator {
+    private static final String CLUSTERING_PROPERTY = ".clustering.distance";
+    private static final String LAYER_ID_PROPERTY = ".baselayer.id";
+    private static final String DEFAULT_RENDER_MODE = "vector";
+
+    public static int migrateStyles (Connection conn, final String layerTable, final String styleTable, final String styleIdColumn) throws SQLException {
+        Map<Long,String> options = getOptions(conn, styleTable);
```

Comment:
Readability of this file would benefit from re-arranging the method order. Maybe move migrateStyles after updateBaseLayerOptions, make updateLayers() private and move it to the end if it's only used from this file?